exporter/export_collection.py

```diff
@@ -17,7 +17,8 @@
 def get_export_filename(export_name, export_type, include_extension=True):
     """Gets a preview of the export filename based on `export_name` and `export_type`."""
     export_name = validate_export_name(export_name)
-    extension = f".{constants.EXPORT_FILE_TYPES[export_type].lower()}" if include_extension else ""
+    file_types = constants.EXPORT_FILE_TYPES_GLTF if get_preferences().use_gltf else constants.EXPORT_FILE_TYPES
```

Comment:
Similarly, if we had set up the `EXPORT_FILE_TYPES` to read directly from the preferences as an enum value (as mentioned above on _constants.py_), we wouldn't need to change any code here.